app/views/jobs/show.html.erb

```diff
@@ -1,38 +1,95 @@
-<h1> <%= @job.job_title %>  (job owned by <%= @job.company.user.email%> )</h1>
-
-<p><%= @job.job_description %> </p>
-
-<p>Company: <%= @job.company.name %></p>
-
-<!-- Apply button only showing to developers -->
-<% if @developer_application %>
-  <p>You have already applied to this job. View your <%= link_to 'application', application_path(@developer_application) %></p>
-<% else %>
-  <% if current_user && current_user.developer %>
-    <%= link_to 'Apply now', new_job_application_path(@job) %>
-  <% elsif !current_user %>
-    <p>Please <%= link_to "login", new_user_session_path %> or create an account to apply for this job</p>
-  <% end %>
-<% end %>
-
-<br>
-
-<!-- pundit needs to be set up here -->
-<% if policy(@job).edit? %>
-  <%= link_to 'edit job posting', edit_job_path(@job) %>
-<% end %>
-<br>
-<% if policy(@job).destroy? %>
-  <%= link_to 'delete job posting', job_path(@job), method: :delete %>
-<% end %>
-<!-- only showing application questions and edit/delete if the company owns the jobs posting -->
-<%# if current_user && current_user == @job.company.user %>
-<% if policy(@job).edit? %>
-  <h3>Application questions</h3>
-  <%= link_to 'Add new question', new_job_question_path(@job) %>
-  <ul>
-    <% @questions.each do |question| %>
-      <li> <%= question.content %> <%= link_to 'edit', edit_question_path(question) %> <%= link_to 'delete', question_path(question), method: :delete %></li>
+<div class="container my-5">
+  <div class="title-container">
+    <div class="title-container-left">
+      <h1>Position: <%= @job.job_title %></h1>
```

Comment:
The way the new seed is written, it's more like "CSS developer wanted". So please remove the "Position", as it won't flow right. Job title can stand alone